charts/logan/values.yaml

```diff
@@ -164,6 +164,8 @@ fluentd:
    # -- To enable MultiProcessWorkers, set this to (> 0) the number of required workers. Defaults to 0.
    # -- Set corresponding workerId using worker field against each source/log. When enabled MultiProcessWorkers, the default worker would be 0 until unless set for each source/log.
    multiProcessWorkers: 0
+   # -- To set timezone for all logs collected via fluentd agent (applies only to log events without timezone identifiers)
```

Comment:
To set timezone override for all logs collected using this solution (applies only to log records without explicit timezone identifier in the record itself)